terraform/a2ha-terraform/variables_common.tf

```
@@ -38,6 +38,11 @@ variable "automate_fqdn" {
   description = "Automate FQDN variable. Leave as an empty string for AWS"
 }
 
+variable "automate_frontend_urls" {
```

why is this value plural ? 
```suggestion
variable "automate_frontend_url" {
```

---

In terraform where ever we are using this variable it was already used as "automate_frontend_urls",if you want me to change as "automate_frontend_url" then i need to change in the existing files as well if thats ok I can change it.

---

Yes please replace it **carefully**.

---

We are sure that it only stores one value and not multiple values?